test/iron-menubar-behavior.html

```diff
@@ -96,6 +109,43 @@
           }, 200);
         });
 
+        suite('left / right keys are reversed when the menubar has RTL directionality', function() {
+          var LEFT = 37;
+          var RIGHT = 39;
+
+          test('left key moves to the next item', function(done) {
+            var rtlContainer = fixture('rtl');
+            var menubar = rtlContainer.querySelector('test-menubar');
+            menubar.selected = 0;
+            menubar.items[1].click();
+            setTimeout(function() {
+              MockInteractions.pressAndReleaseKeyOn(menubar, LEFT);
+              setTimeout(function() {
+                assert.equal(document.activeElement, menubar.items[2], 'document.activeElement is next item');
+                done();
+              // wait for async in _onFocus
+              }, 200);
+            // wait for async in _onFocus
+            }, 200);
+          });
+
+          test('right key moves to the previous item', function(done) {
+            var rtlContainer = fixture('rtl');
+            var menubar = rtlContainer.querySelector('test-menubar');
+            menubar.selected = 0;
+            menubar.items[1].click();
+            setTimeout(function() {
+              MockInteractions.pressAndReleaseKeyOn(menubar, RIGHT);
+              setTimeout(function() {
+                assert.equal(document.activeElement, menubar.items[0], 'document.activeElement is previous item');
```

Comment:
Oh wait, actually no, [this section in WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/#menu) says arrow keys just shift focus. I'll still add a test, but `menubar.selected` should remain 1.